drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/oracle/KogitoAsyncPackageDataModelOracle.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.drools.workbench.screens.scenariosimulation.kogito.client.fakes;
+package org.drools.workbench.screens.scenariosimulation.kogito.client.oracle;
```

Comment:
I would prefer `dmo` as package instead of `oracle`. Wdyt?

Reply:
@danielezonca Ok!